frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -859,6 +860,10 @@ func downloadArtifactToFile(buildAPI BuildAPI, filename, artifactName, buildID,
 
 // Acloud only works with the first instance: cvd-1
 func runAcloudSetup(runtimesRootDir string) {
+	// Do not run this code in unit tests.
```

Comment:
why not? this code is meant to run in production, so it should run in tests as well. If you don't want to modify the host you can always stub/mock/fake it.